style.css

```diff
@@ -553,69 +556,302 @@ textarea {
   font-size: 22px;
 }
 
-@media (max-width: 768px) {
-  .card-body {
-    overflow-x: hidden;
+@media (min-width: 768px) {
+  .first {
+    display: flex;
+    justify-content: center;
+    width: 100vw;
+    min-height: 8vh;
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: transparent;
   }
 
-  .display-centered {
+  .first-img {
     display: none;
   }
 
-  .childs p {
-    padding-left: 20px;
-    font-size: 16px;
+  .second {
+    background-image: url(./images/Header-llustration-desktop.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    gap: 0;
+    width: 100vw;
+    height: 100vh;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .third {
+    font-style: normal;
+    font-weight: 400;
+    text-align: center;
+    line-height: 28px;
+    font-size: 20px;
+    vertical-align: top;
+    color: #172b4d;
+  }
+
+  .green {
+    color: #36b37e;
   }
 
-  .mobile-buttons {
+  .second h1 {
+    margin: 0 0 24px 0;
+  }
+
+  .header-first {
     display: flex;
+    justify-content: center;
+    gap: 50px;
+    font-size: 15px;
+    font-family: 'Inter', sans-serif;
   }
 
-  .asscories .card-tags {
+  .second-text {
+    padding: 10px 0;
+    font-size: 20px;
     display: flex;
-    justify-content: flex-start;
-    width: 100%;
+    justify-content: center;
+    width: 50%;
+    margin: 0 auto;
+  }
+
+  .socials {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    align-items: flex-end;
+    list-style: none;
+    gap: 20px;
+  }
+
+  .lang-text {
+    display: flex;
+    justify-content: center;
+    font-size: 24px;
+    font-family: 'Inter', sans-serif;
+  }
+
+  .lang-text-list {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
     flex-wrap: wrap;
+    gap: 8px;
+  }
+
+  .last {
+    font-family: 'Inter', sans-serif;
+    font-style: regular;
+    font-size: 20px;
+    height: 189px;
+    font-weight: 400;
+    display: flex;
+    justify-content: center;
+    line-height: 32px;
+    margin-top: 17px;
+  }
+
+  .socials-list {
+    width: 30px;
+    height: 30px;
+  }
+
+  .disabled {
+    height: 40px;
+    width: 40px;
+  }
+
+  .recent {
+    display: flex;
+    justify-content: center;
+    font-size: 40px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+  }
+
+  .project-container {
+    display: flex;
+    padding: 30px 30px;
+  }
+
+  .design-1 {
+    gap: 32px;
+  }
+
+  .project {
+    border: 1px solid #c1c7d0;
     margin: 0;
+    width: 369px;
     padding: 0;
-    margin-left: 20px;
+    border-radius: 8px;
+    height: 474px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 16px;
+    margin-bottom: 35px;
   }
```

Comment:
- [ ] [OPTIONAL] To ensure that your project looks more professional, you should stick to the Figma file 👌 . Please make sure that the  `width` is slightly bigger than the `height` of the card 😎 .